src/utils/command.rs

```diff
@@ -324,24 +324,20 @@ where
 pub fn parse_command_with_prefix<'a, N>(
     prefix: &str,
     text: &'a str,
-    bot_name: N,
+    bot_usernamme: N,
 ) -> Option<(&'a str, Vec<&'a str>)>
 where
     N: AsRef<str>,
 {
-    if !text.starts_with(prefix) {
-        return None;
-    }
     let mut words = text.split_whitespace();
     let mut splited = words.next()?[prefix.len()..].split('@');
```

Comment:
Why you've removed this `starts_with`? If the text doesn't start with the prefix, there is no reason to proceed forward. Also, if the text doesn't start with the prefix, `[prefix.len()..]` [can fail](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=4b1aa5fd8f9bb8fb1c198601001d149f).